tests/functional/u/unused/unused_variable.py

```diff
@@ -66,7 +66,7 @@ def hello(arg):
         return True
     raise Exception
 
-# pylint: disable=redefined-outer-name, wrong-import-position,misplaced-future
+# pylint: disable=wrong-import-position,misplaced-future
```

Comment:
I thought it was worth surfacing the `redefined-outer-name` in these tests so you can more easily see that _some_ warning is raised for the fishiness.